lifetimes/fitters/pareto_nbd_fitter.py

```diff
@@ -147,9 +150,19 @@ def conditional_probability_alive(self, frequency, recency, T):
         x, t_x = frequency, recency
         r, alpha, s, beta = self._unload_params('r', 'alpha', 's', 'beta')
 
-        A_0 = np.exp(self._log_A_0([r, alpha, s, beta], x, t_x, T))
-        return 1. / (1. + (s / (r + s + x)) *
-                     (alpha + T) ** (r + x) * (beta + T) ** s * A_0)
+        A_0 = 0
+        if alpha >= beta:
```

Comment:
Heh, I can see I'm not being very pythonic here in contrast with `_log_A_0`, https://github.com/CamDavidsonPilon/lifetimes/blob/227b64a35f9006c154d110a68b3268fc89ad1977/lifetimes/fitters/pareto_nbd_fitter.py#L86. I'm gonna give that a shot.